frontend/lib/uast/NumericLiteral.cpp

```diff
@@ -25,6 +25,16 @@ namespace chpl {
 namespace uast {
 
 
+/*template void NumericLiteral<double, types::RealParam>::dumpFieldsInner(const DumpSettings& s) const {
```

Comment:
Will these be used in the future or is there another reason to leave the commented block here?

Reply:
I ended up leaving it implemented in the header instead. Less code duplication that way. I'll remove these lines, thanks for pointing out the issue.